lib/action_policy/policy/core.rb

```diff
@@ -20,10 +20,10 @@ class UnknownRule < Error
 
     attr_reader :policy, :rule, :message
 
-    def initialize(policy, rule)
+    def initialize(policy, rule, message = nil)
       @policy = policy.class
       @rule = rule
-      @message =
+      @message = message ||
```

Comment:
That's what I told about yesterday: I'd like to have the suggest part in the predicate failure message as well.

So, probably, adding an error subclass instead of extending the existing one would make sense (i.e., `class NonPredicateRule < UnknownRule`).

Reply:
check it, if it's ok I will squash commits